Makefile

```diff
@@ -41,6 +41,10 @@ rapidwright:
 	cd third_party/RapidWright/interchange && make
 	cd third_party/RapidWright && make
 
+# Grab a binary release of rapidwright to use with Python/JPype scripts
+rapidwright_bin:
+	cd third_party && wget https://github.com/Xilinx/RapidWright/releases/download/v2022.2.1-beta/rapidwright-2022.2.1-standalone-lin64.jar
+
```

Comment:
Should this replace the current rapidwright target?  Should we just use the binaries?

Reply:
...I don't know.  The rapidwright we clone has the interchange project within it that we build separately.  I don't know if you get that if you download the binary.  Likewise I don't know how to run rapidwright via Python with the cloned source and not a precompiled jar....

Reply:
You should just do pip install rapidwright. See my comments below.

Reply:
if we do pip install rapdwright, do we still get the interchange project?

Reply:
I've updated to use our existing rapidwright code.